explorer/app/apis/azul/common/filterTransformer.ts

```diff
@@ -15,7 +14,8 @@ import { FilterValue } from "react-table";
 export function transformFilters(filters: Filters): string | undefined {
   // Build up model of filter params from filters.
   const initialFilterParams: {
-    [k: string]: { [key in AZUL_FILTER_OPERATOR]?: FilterValue };
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any -- We can't determine the filter value
+    [k: string]: { [key in AZUL_FILTER_OPERATOR]?: any };
```

Comment:
Can you provide some background why this is updated to `any`? Does react-table no longer expose a type for their filter values?

Reply:
It doesn't. I've looked into the [Filter API](https://tanstack.com/table/v8/docs/api/features/filters) and all related to filter value are typed as `any` or `unkown`.  Also, on v7 `FilterValue` was just `type = any`